src/app/inventory/item-move-service.ts

```diff
@@ -223,15 +223,18 @@ function searchForSimilarItem(
 }
 
 /**
- * Bulk equip items. Only use for multiple equips at once.
+ * Bulk equip items. Only use for multiple equips at once (just loadouts).
+ * Returns a list of items that were successfully equipped, which can be less
```

Comment:
```suggestion
 * Returns a map of items id's to their success status (PlatformErrorCodes.Success if it succeeded), which can be less
```

Reply:
`PlatformErrorCodes` is an odd name for an enum that can contain a `success` state.